src/main-client.ts

```diff
@@ -3303,6 +3323,37 @@ export class MainClient extends BaseRestClient {
     return this.getPrivate('sapi/v1/broker/info');
   }
 
+  updateBNBBurnSubAccount(params: {
+    subAccountId: string;
+    spotBNBBurn: 'true' | 'false';
+  }): Promise<{
+    subAccountId: string;
+    spotBNBBurn: boolean;
+  }> {
+    return this.postPrivate('sapi/v1/broker/subAccount/bnbBurn/spot', params);
+  }
+
+  updateBNBBurnSubAccountMarginInterest(params: {
+    subAccountId: string;
+    interestBNBBurn: 'true' | 'false';
+  }): Promise<{
+    subAccountId: string;
+    interestBNBBurn: boolean;
+  }> {
+    return this.postPrivate(
+      'sapi/v1/broker/subAccount/bnbBurn/marginInterest',
+      params,
+    );
+  }
+
+  getBNBBurnStatusSubAccount(params: { subAccountId: string }): Promise<{
+    subAccountId: string;
+    spotBNBBurn: boolean;
+    interestBNBBurn: boolean;
+  }> {
+    return this.getPrivate('sapi/v1/broker/subAccount/bnbBurn/status', params);
+  }
+
```

Comment:
Reverse the last components of the wording please:
- updateSubAccountBNBBurn
- updateSubAccountMarginInterestBNBBurn
- getSubAccountBNBBurnStatus

Reply:
Done.